crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -596,6 +596,35 @@ type MoveFunction {
 	return: [OpenMoveType!]
 }
 
+type MoveFunctionConnection {
+	"""
+	Information to aid in pagination.
+	"""
+	pageInfo: PageInfo!
+	"""
+	A list of edges.
+	"""
+	edges: [MoveFunctionEdge!]!
+	"""
+	A list of nodes.
+	"""
+	nodes: [MoveFunction!]!
+}
+
+"""
+An edge in a connection.
+"""
+type MoveFunctionEdge {
+	"""
+	The item at the end of the edge
+	"""
+	node: MoveFunction!
+	"""
+	A cursor for use in pagination
```

Comment:
ditto ^, add `.` at the end of the sentence.

Reply:
ditto.